bench/wtperf/wtperf.h

```diff
@@ -127,6 +127,15 @@ struct __truncate_queue_entry {
 };
 typedef struct __truncate_queue_entry TRUNCATE_QUEUE_ENTRY;
 
+/* Steering for the throttle configuration */
+struct __throttle_config {
+	struct timespec last_increment;
+	uint64_t ticket_queue;
```

Comment:
Can you call it something other than `_queue`?  Maybe `_count` since that is also what is used in the workload.  To me that means a TAILQ-style implementation.  Also I find `ticket` confusing.  Can you call it `operations` or `ops` instead?  I suggest `ops_count` and `ops_per_increment`.